decidim-accountability/config/locales/en.yml

```
@@ -257,6 +257,12 @@ en:
             label: Scope
           status_id_eq:
             label: Status
+      titles:
+          new_result: New result
+          statuses: Statuses
+          new_status: New status
+          new_CSV_import: New CSV import
```

I suggested changing the key, and we should also change the value for consistency with the view
```suggestion
          import_results: Import results from CSV
```